File: 2020-10-25--android-p02/app/src/main/java/com/dkass/canvasstuff/MainActivity.java
package com.dkass.canvasstuff;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.dkass.canvasstuff.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        binding.buttonRect.setOnClickListener((View) -> {
            Intent myIntent = new Intent(MainActivity.this, RectActivity.class);
            MainActivity.this.startActivity(myIntent);
        });

        binding.buttonTriangle.setOnClickListener((View) -> {
            Intent myIntent = new Intent(MainActivity.this, TriangleActivity.class);
            MainActivity.this.startActivity(myIntent);
        });

        binding.buttonText.setOnClickListener((View) -> {
            Intent myIntent = new Intent(MainActivity.this, TextActivity.class);
            MainActivity.this.startActivity(myIntent);
        });

        binding.button2.setOnClickListener((View) -> {
            Intent myIntent = new Intent(MainActivity.this, CircleActivity.class);
            MainActivity.this.startActivity(myIntent);
        });
    }

}

File: 2020-10-18--android-z02/z1/app/src/main/java/com/dkass/businesscard/MainActivity.kt
package com.dkass.businesscard

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonShort.setOnClickListener {
            val intent = Intent(this, ShortActivity::class.java)
            startActivity(intent)
        }

        buttonLong.setOnClickListener {
            val intent = Intent(this, LongActivity::class.java)
            startActivity(intent)
        }

        buttonExit.setOnClickListener {
            finishAffinity()
        }
    }


}